src/UI/Areas/Identity/Pages/V3/_Layout.cshtml

```diff
@@ -14,10 +14,10 @@
         <link rel="stylesheet" href="~/Identity/css/site.css" />
     </environment>
     <environment exclude="Development">
-        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.7/css/bootstrap.min.css"
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css"
               asp-fallback-href="~/Identity/lib/bootstrap/dist/css/bootstrap.min.css"
               asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
-        <link rel="stylesheet" href="~/Identity/css/site.min.css" asp-append-version="true" />
+        <link rel="stylesheet" href="~/Identity/css/site.css" asp-append-version="true" />
```

Comment:
This strikes me as wrong. The CSS inside the production library should be the minified version of the code.

Reply:
We discussed this [here](https://github.com/aspnet/Templating/pull/729). We decided that without an out-of-the-box way to minify site.min.* on build it was more dangerous than it was helpful to include those files which can quickly get out of sync with their non-minified versions.

In an update I've removed the site.min.* files, the same as we did in templating.

Reply:
Sure